### PR TITLE
Remove usage tracking

### PR DESCRIPTION
**Pull request by ReedOnly**
closes https://github.com/equinor/isc-appsec/issues/671

### CODE REVIEW

File: docs/about/privacy.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Privacy Policy
+This site do not use any cookies or other tools to track user behavior. Server metrics such as number of HTTP requests will be logged, but no personal information is stored.
```
Comment (larskaare) [inline docs/about/privacy.md:2]: Change last part to "logged. No PII is collected."?

File: docs/about/privacy.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Privacy Policy
+This site do not use any cookies or other tools to track user behavior. Server metrics such as number of HTTP requests will be logged, but no Personal Identifiable Information (PII) is collected.
```
Comment (nilsml) [inline docs/about/privacy.md:2]: Simple and easy to understad. Except from a typo (do should be does) I think it looks good. Of course I am not a lawyer, but if you don't use cookies or track, you should be fine.

**Review by nilsml — CHANGES_REQUESTED** [review]
Fix EDS cdn url and typo

File: docs/about/privacy.md (hunk 2 above)
Comment (ReedOnly) [reply docs/about/privacy.md:2]: great, thanks!

**Review by ipsitamishra16893 — APPROVED** [review]

**Pull request merged**